pkg/build/builder/docker_test.go

```diff
@@ -0,0 +1,87 @@
+package builder
+
+import (
+	"testing"
+)
+
+func TestReplaceValidCmd(t *testing.T) {
```

Comment:
It may be useful but it's out of scope of the current changes ie. command replacement vs Dockerfile validation